src/examples/order/cancel.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Mercado Pago Process Order.
+ *
+ * @see {@link [TODO: insert Order documentation URL] Documentation }.
+  */
+
+import { Order } from '@src/clients/order';
+import MercadoPago from '@src/index';
+
+const mercadoPagoConfig = new MercadoPago({ accessToken: '<ACCESS_TOKEN>', options: { timeout: 5000 } });
+
+const order = new Order(mercadoPagoConfig);
+
+/**
+ * Creates an order and returns its ID.
+ * @returns {Promise<string>} 
+ */
```

Comment:
Podemos simplificar esse comentário, como estamos dentro do exemplo, não há necessidade de usar a anotação `@returns`
```suggestion
// Creates an Order and returns its ID.
```